step/step.go

```diff
@@ -34,6 +34,9 @@ var paths = []string{
 
 	// Configuration cache in project-level folder
 	".gradle/configuration-cache",
+	
+	// JDKs downloaded by the toolchain support
+	".gradle/jdks",
```

Comment:
Isn't this in the Gradle user home folder? According to the [official docs](https://docs.gradle.org/current/userguide/toolchains.html):

> By default, Gradle downloads an Adoptium or AdoptOpenJDK that matches the current operating system and architecture. Gradle installs these JDKs in the [Gradle User Home directory](https://docs.gradle.org/current/userguide/directory_layout.html#dir:gradle_user_home).

Reply:
Yep, youre' right. My bad.
I just copied line 36 from the configuration-cache 😅